src/types/meetings.ts

```diff
@@ -1,43 +1,28 @@
-export interface MeetingSchedule {
-  day: Date;
-  hours: Date[];
-}
+export type AvailablePlatforms = 'google-meet' | 'teams'; // revisit this
 
-export interface MeetingItem {
-  name: string;
-  duration: string;
-  fee: string;
+export interface ScheduleItem {
   id: number;
-  appendedUrlName: string;
-  schedules: MeetingSchedule[];
-}
-
-export interface AvailableHours {
-  // will revisit this interface
-  hour: Date;
-}
-
-export interface Timezone {
-  // will revisit this interface
-  name: string;
+  eventId: number;
+  date: string;
+  isExpired?: boolean;
```

Comment:
Do we want to keep this as nullable?

Reply:
I think this can be non optional(we can merge it and I'll update it in the next PR

Reply:
Okay